app/models.py

```diff
@@ -0,0 +1,46 @@
+from app import db
+
+class Review(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    user_id = db.Column(db.Integer, db.ForeignKey('user.id'),nullable=False)
+    property_id = db.Column(db.Integer, db.ForeignKey('property.id'), nullable=False)
+    estate_agent_id = db.Column(db.Integer, db.ForeignKey('estate_agent.id'))
```

Comment:
lgtm. will we have a column for landlord?

Reply:
i believe not.